api-extractor/src/PrettyPrinter.ts

```diff
@@ -10,23 +10,36 @@ export default class PrettyPrinter {
     * Used for debugging only.  This dumps the TypeScript Compiler's abstract syntax tree.
     */
   public static dumpTree(node: ts.Node, indent: string = ''): void {
-    if (node.kind === ts.SyntaxKind.FromKeyword) {
-      console.log('**');
-    }
+    try {
+      const jsdoc: string = TypeScriptHelpers.getJsDocComments(node, console.log);
+      if (jsdoc && jsdoc.trim()) {
+        console.log('COMMENT=' + jsdoc);
+      }
+    } catch (e) {} /* tslint:disable-line:no-empty */
 
     const kindName: string = ts.SyntaxKind[node.kind];
-    let trimmedText: string = node.getText()
-      .replace(/[\r\n]/g, ' ')
-      .replace(/\s+/g, ' ')
-      .trim();
+    let trimmedText: string;
+
+    try {
+      trimmedText = node.getText()
+        .replace(/[\r\n]/g, ' ')
+        .replace(/\s+/g, ' ')
+        .trim();
 
-    if (trimmedText.length > 100) {
-      trimmedText = trimmedText.substr(0, 97) + '...';
+      if (trimmedText.length > 100) {
+        trimmedText = trimmedText.substr(0, 97) + '...';
+      }
+    } catch (e) {
+      trimmedText = '(error getting text)';
     }
 
     console.log(`${indent}${kindName}: [${trimmedText}]`);
-    for (const childNode of node.getChildren()) {
-      PrettyPrinter.dumpTree(childNode, indent + '  ');
+    try {
```

Comment:
Keep this try/catch block though; it was a fix for a legitimate issue.  The "expected exception" here is okay, since the function is only used for debugging.